src/main/scala/com/fulcrumgenomics/vcf/filtration/ReadEndSomaticVariantFilter.scala

```diff
@@ -123,27 +122,105 @@ class EndRepairArtifactLikelihoodFilter(val distance: Int = 2,
 
     Map(Info.id -> pMutation)
   }
+}
+
+/** The companion object for [[ReadEndSomaticVariantFilter]]. */
+object ReadEndSomaticVariantFilter {
+
+  /** Returns true if <gt> is a heterozygous SNV with one reference allele, else false */
+  private[filtration] def isSnv(gt: Genotype) : Boolean = gt.isHet && !gt.isHetNonRef && gt.calls.forall(_.value.length == 1)
```

Comment:
What if we tested the alleles themselves instead of relying on the auxiliary genotype call info? Pseudo-code:

```scala
def isSnv(gt: Genotype): Boolean = {
    if (gt.alleles.ref.length != 1) false else {
        gt.alleles.alts.collect {
            case alt: SimpleAllele if allele.length == 1 => alt
        }.nonEmpty
    }
}